### DISCUSSION TITLE
Bug in the preview interface when adding long lines of code in a code block of an issue comment in an organisation project

### DISCUSSION THREAD

**Discussion by 0xblank** [community/community#40323] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Hey, I just notice there was a bug in the preview interface when I'm adding long lines of code in a code block of a comment on an issue in an oganisation projet. The code block is way too large. See the image below.
![image](https://user-images.githubusercontent.com/26724501/204418492-5b0e3d9b-af5a-4ce1-b5b1-f53407daec48.png)

For info I use firefox on Debian 12.

**Comment by radar-blip** [comment, marked as answer]
It's a bug for sure.
You can report bugs here if you like: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by 0xblank** [reply]
Just reported it, thanks for the link :+1:
